### PR TITLE
Feat: Refactor Events API

### PR DESCRIPTION
**Pull request by ChenemiKen**
Refactor the Events API
fix issue #50

### CODE REVIEW

File: src/main/java/co/novu/api/events/EventsApi.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Call<TriggerEventResponse> triggerEvent(TriggerEventRequest request);
+
+    @POST(ENDPOINT + "/bulk")
+    Call<BulkTriggerEventResponse> bulkTriggerEvent(BulkTriggerEventRequest request);
+
+    @POST(ENDPOINT + "/broadcast")
+    Call<TriggerEventResponse> broadcastEvent(TriggerEventRequest request);
+
+    @DELETE(ENDPOINT + "/{{transactionId}}")
+    Call<CancelEventResponse> cancelTriggeredEvent(String transactionId);
```
Comment (mayorJAY) [inline src/main/java/co/novu/api/events/EventsApi.java:25]: @Body and @Path annotations are missing

File: src/main/java/co/novu/api/events/EventsApi.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    @POST(ENDPOINT)
+    Call<TriggerEventResponse> triggerEvent(TriggerEventRequest request);
+
+    @POST(ENDPOINT + "/bulk")
+    Call<BulkTriggerEventResponse> bulkTriggerEvent(BulkTriggerEventRequest request);
+
+    @POST(ENDPOINT + "/broadcast")
+    Call<TriggerEventResponse> broadcastEvent(TriggerEventRequest request);
+
+    @DELETE(ENDPOINT + "/{{transactionId}}")
```
Comment (mayorJAY) [inline src/main/java/co/novu/api/events/EventsApi.java:24]: Please use only one pair of curly braces

**Comment by ChenemiKen** [comment]
sorry, my bad

**Comment by ChenemiKen** [comment]
Fixed now.

**Review by mayorJAY — APPROVED** [review]

**Pull request merged**